tests/unit-ext/test-ext-arg.c

```diff
@@ -242,6 +245,52 @@ test_validator2_handler (const jerry_value_t func_obj_val __attribute__((unused)
   return jerry_create_undefined ();
 } /* test_validator2_handler */
 
+/**
+ * The handler should have following arguments:
+ *   arg1: Bool. It is an optional argument.
+ *
+ */
+static jerry_value_t
+test_validator3_handler (const jerry_value_t func_obj_val __attribute__((unused)), /**< function object */
+                         const jerry_value_t this_val, /**< this value */
+                         const jerry_value_t args_p[], /**< arguments list */
+                         const jerry_length_t args_cnt) /**< arguments length */
+{
+
+  bool arg1 = false;
+
+  jerryx_arg_t mapping[] =
+  {
+    /* ignore this */
+    jerryx_arg_ignore (),
+    /* 1th argument should be boolean, and it is optional */
+    jerryx_arg_boolean (&arg1, JERRYX_ARG_COERCE, JERRYX_ARG_OPTIONAL),
```

Comment:
AFAIK, there should be multiple optional args, to see if multiple optional arguments can be handled.

Reply:
Oh and what if an "undefined" value is passed?

Reply:
Good catch! I added tests for it.